x-pack/plugins/apm/kibana.json

```diff
@@ -3,7 +3,15 @@
   "server": true,
   "version": "8.0.0",
   "kibanaVersion": "kibana",
-  "configPath": ["xpack", "apm"],
+  "configPath": [
+    "xpack",
+    "apm"
+  ],
   "ui": false,
-  "requiredPlugins": ["apm_oss", "data", "home"]
+  "requiredPlugins": [
+    "apm_oss",
+    "data",
+    "home",
+    "usageCollection"
```

Comment:
the recommendation is to import this plugin as optional:
https://github.com/elastic/kibana/blob/master/src/plugins/usage_collection/README.md#new-platform

Reply:
done, thanks!